modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -564,8 +566,18 @@ class SegmentEdit extends Component {
 								)}
 							</strong>
 
-							{Liferay.Language.get(
-								'to-enable,-go-to-instance-settings'
+							{this.props.segmentsConfigurationURL ? (
```

Comment:
I'd maybe follow here the pattern of _destructuring_ the props object in line 446 and use the variable directly. Just to have all in one place...